site/blog/airship-update-march-2020.md

```diff
@@ -5,13 +5,13 @@ date: 2020-03-11T12:00:00.000Z
 category: Airship 2.0
 ---
 
-What follows is the Airship Community Update for the month of March, 2020, brought to you by the Airship Technical
+What follows is the Airship Community Update for the month of March 2020, brought to you by the Airship Technical
 Committee.<!-- more -->
 
 ## **SPOTLIGHT ON ....**
 
-- Daylight Saving Time impact on community meetings. If you do not observe Daylight Saving Time please note that meeting
-times are now one hour earlier than they were previously. For the current meeting list and their schedules check the
+- Daylight Saving Time impacts community meetings. If you do not observe Daylight Saving Time, please note that meeting
+times are now one hour earlier than they were previously. For the current meeting list, and their schedules check the
```

Comment:
the comma on L14 should be after schedules instead of list.

Reply:
done.